packages/frontend/tests/integration/components/learner-group/root-test.js

```diff
@@ -104,8 +104,8 @@ module('Integration | Component | learner-group/root', function (hooks) {
     );
     assert.notOk(component.instructorManager.assignedInstructors[1].userNameInfo.hasAdditionalInfo);
     assert.strictEqual(component.associatedCourses.courses.length, 2);
-    assert.strictEqual(component.associatedCourses.courses[0].text, 'course 0');
-    assert.strictEqual(component.associatedCourses.courses[1].text, 'course 1');
+    assert.strictEqual(component.associatedCourses.courses[0].text, 'course 0 (2013)');
```

Comment:
As with instructor group this should get a test for the alternative xxxx - xxxx syntax.